gef.py

```diff
@@ -5210,24 +5219,24 @@ def get_pcustom_filepath_for_structure(self, structure_name):
             raise FileNotFoundError("no file for structure '{}'".format(structure_name))
         return fpath
 
-    def is_valid_struct(self, structure_name):
+    def is_valid_struct(self, structure_name: str) -> bool:
         structure_files = self.enumerate_structures()
         all_structures = set()
         for fname in structure_files:
             all_structures |= structure_files[fname]
         return structure_name in all_structures
 
-    def get_modulename_structname_from_arg(self, arg):
+    def get_modulename_structname_from_arg(self, arg: str) -> Tuple[str, str]:
         modname, structname = arg.split(":", 1) if ":" in arg else (arg, arg)
         structname = structname.split(".", 1)[0] if "." in structname else structname
-        return (modname, structname)
+        return modname, structname
 
-    def deserialize(self, struct, data):
+    def deserialize(self, struct, data: bytes) -> None:
```

Comment:
Do we not have a sane type for `struct`?

Reply:
I couldn't come up with anything when I looked at it. I'm open for suggestions tho

Reply:
Yeah, I guess not. It's an instance of a class that we expect to be a `ctypes.Structure`. We could maybe detect that in `get_structure_class`.

Reply:
so something like this?

```py
def get_structure_class(self, modname: str, classname: str) -> Tuple[Type, ctypes.Structure]:
    ...
```

and 

```py
def deserialize(self, struct: ctypes.Structure, data: bytes) -> None:
    ...
```

Reply:
that seems to be right. i pushed it like this